graphs/0035_GraphNode_Basic.json

```diff
@@ -30,7 +30,7 @@
       "component": "rdf-components/RequestTemplate",
       "metadata": {
         "label": "LOAD INTO GRAPH",
-        "x": 324,
+        "x": 288,
```

Comment:
Yes.  Though I wouldn't lose this graph.  It fits well with the rdf-pipeline ported tests.   Would like to see it put with them so someone looking at the others sees this one too.